lxd/instance_post.go

```diff
@@ -956,7 +957,25 @@ func migrateInstance(s *state.State, r *http.Request, inst instance.Instance, ta
 		return err
 	}
 
-	// Check if we are migrating a ceph-based instance.
+	// In case of live migration, ensure all instance disks can be migrated.
+	if req.Live && inst.IsRunning() {
+		volatileGet := func() map[string]string { return map[string]string{} }
+		volatileSet := func(_ map[string]string) error { return nil }
+		for deviceName, rawConfig := range inst.ExpandedDevices() {
+			dev, err := device.New(inst, s, deviceName, rawConfig, volatileGet, volatileSet)
```

Comment:
Is there any need for device.New here, can we just use rawConfig and pass to IsRootDisk directly?

Reply:
No. I've fixed it, thanks.